src/common/pageMeta/PageMeta.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useRef } from 'react';
+import { Helmet } from 'react-helmet';
+import { useTranslation } from 'react-i18next';
+
+type Props = {
+  title: string;
+};
+
+function PageMeta(props: Props): React.ReactElement {
+  const { title } = props;
+  const { i18n } = useTranslation();
+  const scrollRef = useRef<boolean>(true);
+
+  useEffect(() => {
+    if (scrollRef.current) {
+      window.scrollTo(0, 0);
+      scrollRef.current = false;
+    }
+  }, [scrollRef]);
```

Comment:
When user navigates from page to page, the scroll position does not change. Usually not a problem, because navigation is done from the header. But, for example, in the mobile view after clicking "Saavutettavuusseloste", the content changes, but is shown midway or worse.

Reply:
Not a Firefox issue. The component did not fully re-render on subpage change, so the useEffect was not run. Now it runs every title changes.

Reply:
Works now. But not sure if this is a good feature _for every link_. For example, scroll just a bit down, click another tab (or e.g. change UI language) -> position jumps to top. Feels very weird.

Reply:
In the accessibility statement case this could be a useful feature, but perhaps not for every link.

Reply:
But in non-SPA pages, the jump would happen anyway when you navigate from page to page? The feature in not link related, it happens when page changes.

Reply:
Hmm, that seems to be true. For example this Github PR page has a similar feature: when one clicks on the tabs, the position jumps to the top. I hadn't really noticed that before :astonished: Perhaps it's quite ok then.